{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -100,17 +100,6 @@
 AWS_AUTO_CREATE_BUCKET = True
 AWS_QUERYSTRING_AUTH = False
 
-# AWS cache settings, don't change unless you know what you're doing:
-AWS_EXPIRY = 60 * 60 * 24 * 7
-
-# TODO See: https://github.com/jschneier/django-storages/issues/47
-# Revert the following and use str after the above-mentioned bug is fixed in
-# either django-storage-redux or boto
-control = 'max-age=%d, s-maxage=%d, must-revalidate' % (AWS_EXPIRY, AWS_EXPIRY)
-AWS_HEADERS = {
-    'Cache-Control': bytes(control, encoding='latin-1')
-}
```

Comment:
Looks like this should be replaced by [`AWS_S3_OBJECT_PARAMETERS`](http://django-storages.readthedocs.io/en/latest/backends/amazon-S3.html#settings) instead? It is [empty by default](https://github.com/jschneier/django-storages/blob/06cf8209bf87524c51988ab8e72c8cf97eaa5b89/storages/backends/s3boto3.py#L185).